### ISSUE TITLE
Sorting by multiple people on Beatmap's Discussion page

### ISSUE THREAD

**Issue by Toayhel** [ppy/osu-web#11666]
Hi! Found out how to use GitHub, so I'm basically reposting what I wrote here: https://osu.ppy.sh/community/forums/topics/2004400?n=1

I'd like to suggest a QOL idea of adding a feature that improves sorting on Beatmap's Discussion page.

There are maps that had been modded by a lot of people and when you're a BN it might be hard (more like annoying) to check stuff like

https://toayhel.s-ul.eu/1agllXIX or https://toayhel.s-ul.eu/dCFt38FI

Instead of looking for your own or other BN's mods, you can use "sort by user" thing. However, when you want to nominate a beatmap, you need to do a lot of indepth checks of a beatmap, so it could get ranked without any problems. After a map is modded by both BNs and mods are applied, you need to recheck it and see if your suggestions/problem stamps are properly fixed.

It's not mandatory, but checking other BN's mods wouldn't hurt because you may find something that you didn't notice yourself, but mapper rejected it and you agree with the mod. It would make rechecking a little easier since you don't have to double check the map:

1. Rechecking your mods first and going through the set
2. Rechecking other BN's mod and going through the set again
Instead you could just sort by two or more users, so you could recheck everything at once.

Some BNs expressed their support to this idea on BN Server (may check out the replies below: https://discord.com/channels/316154420591067136/316154420591067136/1307673613390053520), in DMs (https://toayhel.s-ul.eu/XPzk1lcC) and promoted with priority stars on osu! forum I've linked in the first line. That's why it would be amazing if this could be added. I believe that this may help reducing the amount of disqualifications and unranks since rechecking will be a lot easier and take less time (since it may be kinda tedious to recheck a map you've spent a lot of time to mod on)

**Comment by notbakaneko** [comment]
Do you mean being able to select multiple users when filtering?

**Comment by Toayhel** [comment]
Yes, so people could choose not only themselves here, but other specific people too (sort by me and Spkz in this case) 

![image](https://github.com/user-attachments/assets/ccaa144e-5395-4a53-9e18-0fcb48b92a84)
